### ISSUE TITLE
bug(mac): drag and drop of .kmp to configuration window installs silently

### ISSUE THREAD

**Issue by sgschantz** [keymanapp/keyman#12338]
### Describe the bug

When dragging and dropping a .kmp file into the Keyman Configuration window it installs silently. 

### Expected behavior

After dropping a .kmp file on the Keyman Configuration window, Keyman should present a dialog to confirm that the user wants to install the keyboard. If the user agrees and the keyboard is installed successfully, then Keyman should display the Package Information window to provide details about the keyboard that was just installed.

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [X] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

**Comment by mcdurdin** [comment]
On Windows, the standard behaviour is to show the package information dialog (metadata+readme) prior to install, then Welcome dialog after install.

**Comment by sgschantz** [comment]
@mcdurdin I was expecting the behavior to be consistent with what happens when a keyboard is installed using the Download Keyboard  button in the Configuration window and plan on making it conform to that. Does that sound reasonable?

**Comment by mcdurdin** [comment]
Yeah, just noting that the behaviour between the two platforms is not the same. In terms of flow, it made sense to show the readme and metadata to the user before installation, as that helps them have the information they need to decide whether to go ahead with installation -- and then the welcome.htm is shown after installation to help the user start to use the keyboard.

So it may be more of a systemic thing to consider changing on macOS?
